src/hotspot/share/code/relocInfo.cpp

```diff
@@ -86,7 +86,7 @@ relocInfo* relocInfo::finish_prefix(short* prefix_limit) {
     return this+1;
   }
   // cannot compact, so just update the count and return the limit pointer
-  (*this) = prefix_relocInfo(plen);   // write new datalen
+  (*this) = prefix_info(plen);       // write new datalen
```

Comment:
Just a minor thing: Any specific reason for these additional whitespaces?

Reply:
No, I've no idea where those spaces came from.  I thought maybe I'd hit `meta-;` (emacs indent for comment), but nope, that puts a different number of spaces there.  And it's not manually lined up with other comments in the function.

Reply:
Okay :-)